src/assets/CollectiblesController.test.ts

```diff
@@ -561,10 +627,10 @@ describe('CollectiblesController', () => {
       ).toStrictEqual([
         {
           address: ERC721_KUDOSADDRESS,
-          description: 'Kudos Description',
```

Comment:
I am not 100% sure why these tests changed after I replaced the Nock mocks. I noticed that some requests were being called for real and I attempted to mock those. It's possible I missed something. If this were not a draft PR, I would correct this, but for the time being, I just tried to get the tests passing.